packages/state-transition/src/block/processAttestations.ts

```diff
@@ -10,7 +10,7 @@ import {processAttestationsAltair} from "./processAttestationsAltair.js";
 export function processAttestations(
   fork: ForkSeq,
   state: CachedBeaconStateAllForks,
-  attestations: phase0.Attestation[],
+  attestations: allForks.Attestation[],
```

Comment:
consider handle `electra.attestations` in a separate file based on `fork >= ForkSeq.electra` as below

Reply:
can reuse altair 👍